ui/src/forms/index.ts

```diff
@@ -48,8 +49,10 @@ renderer.setStrategy({
   object: objectStrategy,
 })
 components.pushComponents(Components)
+components.decorate(instancesSelector.decorator())
 
 editors.addMatchers(Matchers)
+editors.decorate(instancesSelector.matcher)
```

Comment:
`shaperone-hydra` adds a decorator to handle `sh:property > hydra:collection` and a customised matcher to add a higher score to `dash:InstancesSelectEditor` where `hydra:collection` is used